DeAceroDLabs/marketplace-frontend
# Add popup on error with error messages and update success popup

## What is done

[[DMP-790] [Frontend] Mostrar estado de creación de orden](https://deacero.atlassian.net/browse/DMP-790)

## How is done

1. Modified popup on success to match given design
2. Added a popup on error
3. Added some error messages

## How to test

1. Run the project via `npm start`
2. Add at least an item to cart
3. Go to checkout until the last step
4. Fill out the form and submit
5. Popup should show either a success or error message
6. An intentional error could be either not entering a CVV or a 4 digit one, or an invalid date

## Evidence
